wechatpy/client/api/wxa.py

```diff
@@ -44,20 +50,22 @@ def get_wxa_code_unlimited(self,
                                width=430,
                                auto_color=False,
                                line_color={"r": "0", "g": "0", "b": "0"},
-                               page=None):
+                               page=None,
+                               is_hyaline=False):
         """
         创建小程序码（接口B：适用于需要的码数量极多，或仅临时使用的业务场景）
         详情请参考
         https://mp.weixin.qq.com/debug/wxadoc/dev/api/qrcode.html
         """
         return self._post(
             'wxa/getwxacodeunlimit',
-            data=optionaldict(
```

Comment:
I'd prefer not to change this.

Reply:
Could you give me a reason to use `optionaldict` instead of `dict` in this particular place?

Reply:
To avoid having `{"page": null}` in JSON.

Reply:
That makes sense.